api/provenance/provenance.go

```diff
@@ -62,6 +62,13 @@ func GetProvenance() Provenance {
 			p.GitCommit = setting.Value
 		}
 	}
+
+	for _, dep := range info.Deps {
+		if dep.Path == "sigs.k8s.io/kustomize/kustomize" {
```

Comment:
issue: this needs to be
```suggestion
		if dep.Path == "sigs.k8s.io/kustomize/kustomize/v5" {
```

nit (non-blocking): it would be nice to have a constant for the module name with a comment explaining why we are looking for this specific dependency.